pkg/web_app/lib/src/search.dart

```diff
@@ -5,12 +5,33 @@
 import 'dart:html';
 
 void setupSearch() {
+  _setEventForKeyboardShortcut();
   _setEventForSearchInput();
   _setEventForFiltersToggle();
   _setEventForSortControl();
   _setEventForCheckboxChanges();
 }
 
+void _setEventForKeyboardShortcut() {
+  final inputElem = document.querySelector('input.site-header-search-input');
+  if (inputElem != null && inputElem is InputElement) {
+    window.onKeyPress.listen((e) {
+      // Ignore keys other than the shortcut key:
+      if (e.key != '/') return;
+
+      // Only trigger focus when nothing is focused, so that we don't steal
+      // focus from other elements.
+      final active = document.activeElement;
+      if (active == null || active == document.body) {
+        inputElem.focus();
```

Comment:
We could also ask if the current element is not an input / textarea / etc...

I think think there are many elements that take input. But it's easy to give focus to link or other thing, is it not?